Foundation/CGFloat.swift

```diff
@@ -17,6 +17,8 @@ public struct CGFloat {
     /// The native type used to store the CGFloat, which is Float on
     /// 32-bit architectures and Double on 64-bit architectures.
     public typealias NativeType = Double
+#else
+    public typealias NativeType = Float
 #endif
```

Comment:
This works. seems to be a swift bug when an arch is not known:
#if arch(i386) || arch(arm) // || arch(powerpc)